hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyRequest.java

```diff
@@ -78,8 +79,8 @@
   protected OzoneBlockTokenSecretManager ozoneBlockTokenSecretManager;
   protected ScmBlockLocationProtocol scmBlockLocationProtocol;
 
-  protected final long containerID = 1000L;
-  protected final long localID = 100L;
+  protected static final long CONTAINERID = 1000L;
+  protected static final long LOCALID = 100L;
```

Comment:
`CONTAINER_ID` and `LOCAL_ID` would be more in line with naming conventions.

Reply:
Fixed.